src/main/java/org/hisp/dhis/tasks/DhisAbstractTask.java

```diff
@@ -228,4 +231,24 @@ protected void logWarningIfDebugEnabled( String message )
             logger.warning( message );
         }
     }
+
+    protected User getRandomUser( Randomizer rnd ) {
+        return new UserRandomizer(rnd).getRandomUser( entitiesCache );
+    }
+
+    public String getRandomUserOrgUnit( User user, Randomizer rnd )
+    {
+        return new UserRandomizer(rnd).getRandomUserOrgUnit( user );
+    }
+
+    public String getRandomUserOrProgramOrgUnit( User user, Program program, Randomizer rnd )
```

Comment:
do we need to pass in the Randomizer if its a field on this class?

Reply:
That is going to be addressed at a later stage when we clean up the `Task`/`TaskSet` situation